core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -811,20 +811,29 @@ class KRaftClusterTest {
         assertTrue(cluster.controllers.asScala.keySet.contains(quorumInfo.leaderId),
           s"Leader ID ${quorumInfo.leaderId} was not a controller ID.")
 
-        quorumInfo.voters.forEach { voter =>
-          assertTrue(0 < voter.logEndOffset,
-            s"logEndOffset for voter with ID ${voter.replicaId} was ${voter.logEndOffset}")
-          assertNotEquals(OptionalLong.empty(), voter.lastFetchTimestamp)
-          assertNotEquals(OptionalLong.empty(), voter.lastCaughtUpTimestamp)
+        val(voters, voterResponseValid) = TestUtils.computeUntilTrue(
+          admin.describeMetadataQuorum(new DescribeMetadataQuorumOptions)
+            .quorumInfo().get().voters()){
```

Comment:
nit: feels like there should be a space before the brace? I would probably write it like this:
```scala
TestUtils.computeUntilTrue(
  admin.describeMetadataQuorum(new DescribeMetadataQuorumOptions)
    .quorumInfo().get().voters()
) { voters =>
  voters.stream.allMatch { voter => 
    voter.logEndOffset > 0 && 
    voter.lastFetchTimestamp() != OptionalLong.empty() && 
    voter.lastCaughtUpTimestamp() != OptionalLong.empty()
  }
}
```
Guess it still looks a bit awkward...

Reply:
Yeah. The readability is not the best here. I just tried to do something that satisfied the checkstyle TBH :D.
Will include your suggestion.